tests/consumer/test_oauth1.py

```diff
@@ -556,3 +556,23 @@ def test_custom_session_class():
     )
     assert isinstance(bp.session, CustomOAuth1Session)
     assert bp.session.my_attr == "foobar"
+
+
+def test_rule_kwargs():
+    blueprint = OAuth1ConsumerBlueprint(
+        "test-service",
+        __name__,
+        client_key="client_key",
+        client_secret="client_secret",
+        base_url="https://example.com",
+        request_token_url="https://example.com/oauth/request_token",
+        access_token_url="https://example.com/oauth/access_token",
+        authorization_url="https://example.com/oauth/authorize",
+        redirect_to="my_view",
+        rule_kwargs={"host":"example2.com"},
+    )
+    app = flask.Flask(__name__)
+    app.secret_key = "secret"
+    app.register_blueprint(blueprint, url_prefix="/login")
+    rules = [rule for rule in app.url_map.iter_rules() if rule.endpoint.startswith("google.")]
```

Comment:
Please assert that there should be exactly two rules that match. Right now, I believe this test is only passing because _zero_ rules match this filter.

Reply:
Done.  I've done the same for the oauth2, google and jira tests.  The jira test still said "google", so had the same issue.